src/server/models/app-models.js

```diff
@@ -182,6 +185,48 @@ export default function (db) {
 
   const ParticipantAllergy = db.define('participant_allergy')
 
+  const AuditClientData = db.define(
```

Comment:
I'd argue that for audit data, it's perfectly ok for us to just inline this data into the main AuditEvent. We want to have minimal processing of the data for speed and to minimize any risk of errors when processing it.

Reply:
Sure, anything works for me. My brain is just used to optimize things like this.